babel.config.js

```diff
@@ -69,7 +69,23 @@ module.exports = function (api) {
           pragma: 'h',
         },
       ],
-      'inline-react-svg',
+      [
+        'inline-react-svg',
+        {
+          svgo: {
+            plugins: [
+              {
+                name: 'preset-default',
+                params: {
+                  overrides: {
+                    removeViewBox: false,
+                  },
+                },
+              },
+            ],
+          },
+        },
+      ],
```

Comment:
Our icons didn't work properly in Storybook and generally in preact context. This should fix it. SVGO library was removing `viewBox` attribute by default and it was preventing icons from being resized. We never needed resized icons but for pills we do. Hence the update.